packages/transition-frontend/src/components/forms/node/TransitNodeEdit.tsx

```diff
@@ -142,6 +143,43 @@ class TransitNodeEdit extends SaveableObjectForm<Node, NodeFormProps, NodeFormSt
             transitNodes1000mRadius: turfFeatureCollection(geojson.geometry ? [geojson] : []),
             transitNodesRoutingRadius: turfFeatureCollection(geojson.geometry ? [geojson] : [])
         });
+        serviceLocator.socketEventManager.emit(
+            'osm.streetsAroundPoint',
+            this.state.object.toGeojson(),
+            100,
+            (status: Status.Status<GeoJSON.Feature[]>) => {
+                if (Status.isStatusOk(status)) {
+                    const streetsGeojsonFeatures: GeoJSON.Feature[] = status.result;
+                    if (streetsGeojsonFeatures.length < 10) {
```

Comment:
probably need an else (and also a specific test for this condition) with a log to know what is the error if we get it

Reply:
This part is changed, we now check for intersections with turf and order them by distance